app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class SessionsController < Devise::SessionsController
+  def destroy
+    # Let's say goodbye to all the cookies when someone signs out.
+    cookies.each do |cookie|
```

Comment:
should we add a test for this? you can access `response.cookies` in a request spec